streaming/base/shared/prefix.py

```diff
@@ -30,6 +30,19 @@ def _each_prefix_int() -> Iterator[int]:
         prefix_int += 1
 
 
+def _create_filename(prefix_int: int, name: str) -> str:
```

Comment:
are these technically files? maybe call it get_path?

Reply:
Done in https://github.com/mosaicml/streaming/pull/299/commits/56368cb07a7657e6e52aef95fb725404c509e6bf.